### PR TITLE
Update TraceLog.md

### PR DESCRIPTION
**Pull request by wilders-soccerfan**
Thank you for your contribution to the Sandboxie-docs repository.

If several changes need to be submitted in more than one file, it is mandatory to include them all in the same pull request, otherwise the others may be closed.

### CODE REVIEW

**Comment by wilders-soccerfan** [comment]
My first attempt. Hope I'm not in violation of any contributing guidelines. Thank you.

**Review by isaak654 — CHANGES_REQUESTED** [review]
Restore the original line ending (LF) in Notepad++ or other text editor, save the file and reupload it as a new commit in your [fork](https://github.com/wilders-soccerfan/sandboxie-docs). The change will be shown here too.

https://github.com/sandboxie-plus/sandboxie-docs/assets/12372772/f731e3b1-6fb7-4a9a-b913-ccdfe4809d39

**Comment by wilders-soccerfan** [comment]
In Notepad++, I see an encoding tab with ANSI, UTF-8 without BOM, UTF-8 etc. Do I need to select one of these before saving? Which one? Sory, I am confused.

**Comment by isaak654** [comment]
Encodings have nothing to do with line endings.
On the status bar, you should also notice a `Windows (CRLF)` reference to change in LF, you can use the previous video as reference.

I would need this in order to preview your changes correctly.

**Comment by isaak654** [comment]
Encodings have nothing to do with line endings.
On the status bar, you should also notice a `Windows (CR LF)` reference to change in `Unix (LF)`, you can use the previous video as reference.

I would need this in order to preview your changes correctly.

File: PlusContent/TraceLog.md [hunk 1]
```diff
@@ -21,3 +21,9 @@ The Trace Log tool displays the names of any system resources that are accessed
 **Performance Impact**
 
 When inactive, the Trace Log does not use any system resources and does not have any performance impact on any running programs. When active, the Trace Log has a small performance penalty on sandboxed programs.
+
+**Recent Improvements**
+
+**v1.9.6** adds a full stack trace to all trace messages.
+Note that Enable the Trace Logging also turns on the Keep Terminated.
+This is not a bug but new intended behaviour. Without it, the stack trace in the trace log would not work properly, as it uses the process objects to cache the symbols.
```
Comment (isaak654) [inline PlusContent/TraceLog.md]:
```suggestion
**Sandboxie Plus v1.9.6** adds a full stack trace to all trace messages.
Note that activating the Trace Log also turns on the Keep Terminated feature.
This is not a bug, but a new intended behaviour. Without it, the stack trace in the Trace Log would not work properly, as it uses the process objects to cache the symbols.
```

If you agree, please press the `Commit suggestion` -> `Commit changes` button to apply it.

**Review by isaak654 — CHANGES_REQUESTED** [review]

**Comment by isaak654** [comment]
At this point, it would be ideal to merge the three commits into one with a proper title like `Update TraceLog.md`. 
A simple way not to use command lines is included in [this](https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/managing-commits/squashing-commits-in-github-desktop) guide (which requires the installation of GitHub Desktop). 

However, I'm getting used with the `Squash and merge` option that is here for those who have the appropriate permissions, so I can do this right now:
immagine

**Review by isaak654 — APPROVED** [review]

**Comment by isaak654** [comment]
Note: the failed Spell check status consists in new typos not related to this pull request: 
https://github.com/sandboxie-plus/sandboxie-docs/actions/runs/5126332656/jobs/9220682856

typos

It is not necessarily up to me to fix them, so other contributors may do so.

**Pull request merged**